core/src/test/scala/io/snappydata/core/LocalTestData.scala

```diff
@@ -81,6 +81,7 @@ object LocalSparkConf {
     val conf = new SparkConf()
         .setIfMissing("spark.master", "local[4]")
         .setIfMissing("spark.memory.debugFill", "true")
+        .setIfMissing("snappydata.sql.planCaching", "true")
```

Comment:
I think you will also need to set it for dunit tests -- see bootProps/sysProps in ClusterManagerTestBase